Modules/System/Email/src/Email/Outbox/EmailEditor.Page.al

```diff
@@ -59,11 +59,21 @@ page 13 "Email Editor"
                             ToolTip = 'Specifies the email addresses to send the email to.';
                             Editable = not EmailScheduled;
                             Importance = Promoted;
+                            Lookup = true;
 
                             trigger OnValidate()
                             begin
                                 EmailMessage.SetRecipients(Enum::"Email Recipient Type"::"To", ToRecipient);
                             end;
+
+                            trigger OnLookup(var Text: Text): Boolean
+                            var
+                                LookupOk: Boolean;
+                            begin
+                                LookupOk := true;
+                                OnLookupToRecipient(Text, LookupOk);
```

Comment:
it should be possible to write a test or two, one that does a lookup to enter a value and one that doesn't... in both cases the values should be validated... Maybe @GiorgosPa add some more guidance...